schema/site_metadata.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$udmi_version": "1.5.0",
+    "title": "Site Metadata",
+    "$section": "Site Model",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
```

Comment:
should have "additionalProperties": false on the other "object" entries below (so location and origin)